tests/test_fixity.py

```diff
@@ -0,0 +1,472 @@
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+    ],
+)
+def test_scanall(_get, capsys):
+    response = fixity.scanall(
+        ss_url=STORAGE_SERVICE_URL,
+        ss_user=STORAGE_SERVICE_USER,
+        ss_key=STORAGE_SERVICE_KEY,
+        session=SESSION,
+    )
+
+    assert response is True
+
+    captured = capsys.readouterr()
+    assert "Successfully scanned 2 AIPs" in captured.err
+
+
+@mock.patch(
+    "requests.get",
+    side_effect=[
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "meta": {"next": None},
+                    "objects": [
+                        {
+                            "package_type": "AIP",
+                            "status": "UPLOADED",
+                            "uuid": str(uuid.uuid4()),
+                        },
+                        {
+                            "package_type": "AIP",
+                            "status": "UPLOADED",
+                            "uuid": str(uuid.uuid4()),
+                        },
+                    ],
+                },
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 401,
+                "json.return_value": {},
+            },
+            spec=requests.Response,
+            side_effect=Exception,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 401,
+                "json.return_value": {},
+            },
+            spec=requests.Response,
+            side_effect=Exception,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
```

Comment:
Correct me if I'm wrong, but I think this represents a successful call to the `check_fixity` endpoint. Two thoughts about it:
* Since there are many occurrences of the same mock, could it be abstracted to a variable at the top of the module and reused since it only contains static data?
* I don't see any scenario testing `"success": False` with `failures` being populated. How would this package react to that? Should we add a test for it?